src/Storefront/Resources/views/storefront/component/product/properties.html.twig

```diff
@@ -18,7 +18,7 @@
                                                 {% apply spaceless %}
                                                     {% for option in group.options %}
                                                         {% set i = ( i | default(0) ) + 1 %}
-                                                        <span>{% if i > 1 %}, {% endif %}{{ option.translated.name|e }}</span>
+                                                        <span>{% if i > 1 %}, {% endif %}{{ option.translated.name|raw }}</span>
```

Comment:
I am fine, when we use the existing html sanitizer. but I don't see we merge this when we use `|raw` and let the storefront execute JS

Reply:
I used sw_sanitize instead